src/components/MapView/responder/index.ts

```diff
@@ -1,8 +1,3 @@
-import {PanResponder} from 'react-native';
+import SwipeInterceptPanResponder from '../../SwipeInterceptPanResponder';
 
-const responder = PanResponder.create({
-    onMoveShouldSetPanResponder: () => true,
-    onPanResponderTerminationRequest: () => false,
-});
-
-export default responder;
+export default SwipeInterceptPanResponder;
```

Comment:
What's the point of this file if all it does is export another file. Do we need it?

Reply:
My reasoning here was to try to maintain the `responder` file structure since we have platform-specific implementations for web, iOS and Android. It's not "strictly" needed.

Reply:
@puneetlath Looks like I lied. The TypeScript checks job fails if there is no `index.ts` file present, so this is actually needed for now.

Reply:
Ok got it!